StereoKitC/libraries/sk_gpu.h

```diff
@@ -541,6 +541,7 @@ typedef struct skg_platform_data_t {
 ///////////////////////////////////////////
 
 SKG_API void                skg_setup_xlib               (void *dpy, void *vi, void *fbconfig, void *drawable);
+SKG_API bool                skg_egl_dri                  ();
```

Comment:
For a function to be added to sk_gpu's public API, it should be implemented for both GL and D3D. This might make more sense as a capabilities check in `skg_capability` if it must be exposed?